packages/button/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.2.0",
   "main": "dist/main.js",
   "license": "MIT",
+  "typings": "definitions/index.d.ts",
```

Comment:
well template should reflect desired state for each package, also it doesn't have to do anything with our current build if definitions are not working it shouldn't be here as well.

Simply I want to have package.json to be auto generated to some degree so we can easily change tooling for all packages.